loqusdb/build_models/variant.py

```diff
@@ -25,7 +26,7 @@ def check_par(chrom, pos):
     """
     par = False
 
-    for interval in PAR.get(chrom,[]):
+    for interval in PAR[global_parameters.GENOME_BUILD].get(chrom, []):
```

Comment:
```suggestion
    for interval in PAR[build].get(chrom, []):
```
just add `build='37'` to the arguments above